### PR TITLE
Show git prune output

### PR DESCRIPTION
**Pull request by yosefd**
/CC @gilad61 @eladb

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "Yosef Dinerstein",
         "Gilad Oren"
     ],
-    "version": "0.1.2",
+    "version": "0.1.3",
```
Comment (eladb) [inline package.json:11]: Did you publish?

**Comment by eladb** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (yosefd) [inline package.json:11]: Will publish once it will work.

**Comment by gilad61** [comment]
Looks good, thanks!

**Pull request merged**